content/dotnet-template-azure-iot-edge-module/CSharp/Program.cs

```diff
@@ -1,86 +1,10 @@
-namespace SampleModule
-{
-    using Microsoft.Azure.Devices.Client;
-    using Microsoft.Azure.Devices.Client.Transport.Mqtt;
-    using System;
-    using System.Runtime.Loader;
-    using System.Text;
-    using System.Threading;
-    using System.Threading.Tasks;
+using SampleModule;
 
-    class Program
+IHost host = Host.CreateDefaultBuilder(args)
+    .ConfigureServices(services =>
     {
-        static int counter;
+        services.AddHostedService<ModuleBackgroundService>();
+    })

```

Comment:
Since this body just has 1 statement, it can be simplified into 1 line, if you like.

```csharp
    .ConfigureServices(servcies => services.AddHostedService<ModuleBackgroundService>())_
```

Reply:
I just keep the code formatting from the default Worker template.